doc/elements.md

```diff
@@ -78,6 +78,29 @@ Attribute | Type | Default | Description
 `digits` | integer | 2 | number of digits that must be correct for `comparison="sigfig"` or `comparison="decdig"`.
 `eps_digits` | integer | 3 | Additional digits (beyond `digits`) used to compute a grace tolerance.
 
+## `pl_file_upload` element
+
+```html
+<pl_file_upload files="['foo.py','bar.c']" />
+```
+
+Provides a way to accept file uploads as part of an answer. They will be stored
+in [the format expected by externally graded questions](externalGrading.md#file-submission-format).
+
+Attribute | Type | Default | description
+--- | --- | --- | ---
+`answers_name` | string | \_file | Variable name to store data in. **For externally graded questions, you should rely on the default.**
+`filenames` | JSON array | [] | List of files that should and must be submitted.
```

Comment:
Or we just choose a delimiter and don't provide an escaping mechanism. I can't really think of a reason why someone would need to have a file containing a comma, for example. I'd be happy with comma (e.g., CSV), colon (e.g., PATH), or semicolon as a delimiter.